mock/py/mockbuild/util.py

```diff
@@ -582,7 +582,7 @@ def logOutput(fdout, fderr, logger, returnOutput=1, start=0, timeout=0, printOut
                         if line != '':
                             line = ansi_escape.sub('', line)
                             if fderr is s and not line.startswith('+ '):
-                                logger.debug("BUILDSTDERR: %s", line)
+                                logger.debug("%s%s" % (logging.getLogger('mockbuild').stderr_line_prefix, line))
```

Comment:
Why can not we get the `config_opts['stderr_line_prefix']` value here?   It at least looks to be equivalent;  as long as we don't just do `logger.stderr_line_prefix`.

Reply:
@praiskup There is no `config_opts` at this place of code. :(

Reply:
pff, ok, to not complicate this ... what about to "admit" this is hack by comment, and maybe namespace the attribute by `mock_` prefix so the future clash is unlikely....  Btw., why we don't use `logger.stderr_line_prefix`?

Reply:
If one wanted to have a bit more complicated life :-) there should be a way to solve this using logging.Formatter.